demos/select.html

```diff
@@ -97,37 +99,30 @@
 
       <section class="example">
         <h2 class="mdc-typography--title">Fully-Featured Component</h2>
-        <section id="demo-wrapper">
-          <div id="js-select" class="mdc-select" role="listbox" tabindex="0">
-            <span class="mdc-select__selected-text">Pick a food group</span>
+        <section id="box-demo-wrapper">
+
+          <div id="js-select-box" class="mdc-select" role="listbox" tabindex="0">
+            <div class="mdc-select__surface">
+              <div class="mdc-select__label">Food Group</div>
+              <div class="mdc-select__selected-text"></div>
+              <div class="mdc-select__bottom-line"></div>
+            </div>
             <div class="mdc-simple-menu mdc-select__menu">
               <ul class="mdc-list mdc-simple-menu__items">
-                <li class="mdc-list-item" role="option" id="grains" aria-disabled="true">
-                  Pick a food group
+                <li class="mdc-list-item" role="option" id="fruit-roll-ups" tabindex="0">
+                  Fruit Roll Ups
                 </li>
-                <li class="mdc-list-item" role="option" id="grains" tabindex="0">
-                  Bread, Cereal, Rice, and Pasta
+                <li class="mdc-list-item" role="option" id="cotton-candy" tabindex="0">
+                  Candy (cotton)
                 </li>
                 <li class="mdc-list-item" role="option" id="vegetables" aria-disabled="true" tabindex="0">
                   Vegetables
                 </li>
-                <li class="mdc-list-item" role="option" id="fruit" tabindex="0">
```

Comment:
The intention is not about making a group header. It is about a disabled option. The others were removed for the sake of brevity.